config/initializers/timezone.rb

```diff
@@ -0,0 +1,2 @@
+# Set the timezone for the JavascriptAgent (libv8 only relies on the TZ variable)
+ENV['TZ'] = Time.zone.tzinfo.canonical_identifier
```

Comment:
I support that, let's leave this as you have it. LGTM!
On Wed, Mar 16, 2016 at 9:40 AM Dominik Sander <notifications@github.com>
wrote:

> In config/initializers/timezone.rb
> <https://github.com/cantino/huginn/pull/1356#discussion_r56368208>:
>
> > @@ -0,0 +1,2 @@
> > +# Set the timezone for the JavascriptAgent (libv8 only relies on the TZ variable)
> > +ENV['TZ'] = Time.zone.tzinfo.canonical_identifier
>
> My thought was to use the TIMEZONE .env variable as truth for the
> timezone the user selects. If we would callback to the system wide TZ
> when it's present, the only way to have a consistent timezone setting would
> be to add TZ to .env as well.
>
> —
> You are receiving this because you commented.
>
>
> Reply to this email directly or view it on GitHub
> <https://github.com/cantino/huginn/pull/1356/files/c48ff63e7a96260540f9399201cbd7a23d65c9fa#r56368208>
>